tuning/utils/merge_model_utils.py

```diff
@@ -102,3 +105,95 @@ def fetch_base_model_from_checkpoint(checkpoint_model: str) -> str:
             "Base model adapter config exists, but has no base_model_name_or_path!"
         )
     return adapter_dict["base_model_name_or_path"]
+
+
+def copy_files_to_directory(src: str, dest: str, exclude_files: list[str] = None):
```

Comment:
We hit into an issue previously when using copytree ([internal link](https://github.ibm.com/ai-foundation/watson-fm-stack-tracker/issues/1089)) where the permissions and ownership of the directory was being copied which caused problems with error `Operation not permitted'`

Reply:
We do have a similar function [here](https://github.com/foundation-model-stack/fms-hf-tuning/blob/main/build/utils.py#L27) that recursively copies all of the contents of a directory but it doesn't have the exclude and copies the whole directory instead of just the files which is what this method is doing